Solutions/ESI - Exchange Security Config and Audit Insights/Analytic Rules/ServerOrientedWithUserOrientedAdministration.yaml

```diff
@@ -0,0 +1,88 @@
+id: 7bce901b-9bc8-4948-8dfc-8f68878092d5
+name: Server Oriented Cmdlet And User Oriented Cmdlet used
+description: |
+  'Detect if a server oriented Cmdlet and a user oriented cmdlet that are monitored are launched by a same user in a same server in a 10 minutes timeframe'
+requiredDataConnectors:
+  - connectorId: ESI-ExchangeAdminAuditLogEvents
+    dataTypes:
+      - Event
+severity: High
+queryFrequency: 5h
+queryPeriod: 5h
```

Comment:
The query looks back 1 day of data always and compares it against 2 days of data. If this is done every 5 hours, there are chances that this would give duplicate results. Can we improve either the query logic or query frequency and period.

Reply:
If I put a query frequency/period to 1d, is it ok for you ?

Reply:
Frequency and period changed to 1 day.